en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -12,7 +12,7 @@ the server to update templates, website logic, and perform upgrades or maintenan
 
 ## PHP
 
-- PHP 8.3
+- PHP 8.3 or newer
```

Comment:
```suggestion
- PHP 8.3 - 8.4
```

Or to match the way it is in the [CMS 5 docs](https://docs.silverstripe.org/en/5/getting_started/server_requirements/):
```suggestion
- PHP >=8.1, <=8.3
```

8.5 isn't supported in 6.0.0, and won't be until 6.2.0 - and then anything newer that comes out won't be supported for 6.x at all.